### DISCUSSION TITLE
Digital Audio Workstation (DAW)

### DISCUSSION THREAD

**Discussion by glan99** [ExistentialAudio/BlackHole#782] [Q&A]
The examples show a Digital Audio Workstation (DAW).
https://existential.audio/blackhole/support/#micsystem

Which particular DAW is that?  Is it still available?  If not, do you have a particular recommendation for Mac?

Thank you

**Comment by devinroth** [comment]
That is Logic Pro. I also use Cubase.

If you are looking for an affordable yet powerful DAW I would recommend Reaper.

Devin

**Reply by glan99** [reply]
Thank you, and thank you for suggesting an alternative.  I looked and certainly agree that Reaper is affordable.  I also noticed the generous evaluation period.  I think that for my project, Reaper may be an excellent place to start.

For completeness, I am also wondering if any free DAW can be recommended.  I am only beginning to learn about these things and am aware I cannot even ask intelligently.  In the searches I conducted, I noticed Garage Band mentioned as a possibility but know nothing about it other than by name. If no free DAW can be recommended, I would understand.

Thank you again, very much.

**Comment by devinroth** [comment]
GarageBand is fine as a free option but lacks many features. Logic Pro is essentially the pro version of GarageBand. Totally fine if you want to give GarageBand a try.

Devin

**Comment by glan99** [comment]
Thank you very much Devin for your suggestions and descriptions.  So very helpful.  Thank you!
